etc/picongpu/hemera-hzdr/k80.tpl

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env bash
+# Copyright 2013-2019 Axel Huebl, Richard Pausch, Rene Widera, Marco Garten
+#
+# This file is part of PIConGPU.
+#
+# PIConGPU is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# PIConGPU is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with PIConGPU.
+# If not, see <http://www.gnu.org/licenses/>.
+#
+
+
+# PIConGPU batch script for hemera' SLURM batch system
+
+#SBATCH --partition=!TBG_queue
+# necessary to set the account also to the queue name because otherwise access is not allowed at the moment
+#SBATCH --account=!TBG_queue
```

Comment:
The `--account` option is typically used for project allocations but at the moment one has to specify this as well because all users that have access to the k80s are in this "project".

Maybe this can be rather put to the `-C / --constraints` option.

Reply:
In other profiles, we use a `$proj` variable from `picongpu.profile` for that, check out Titan or Taurus for examples :)

Reply:
I think we can stay here with the direct project selection since the name is always `k80` because we have no real projects on this system.

Reply:
I'll still call it `$proj` just for consistency.